toplevel/ccompile.mli

```diff
@@ -0,0 +1,19 @@
+(************************************************************************)
+(*         *   The Coq Proof Assistant / The Coq Development Team       *)
+(*  v      *   INRIA, CNRS and contributors - Copyright 1999-2018       *)
+(* <O___,, *       (see CREDITS file for the list of authors)           *)
+(*   \VV/  **************************************************************)
+(*    //   *    This file is distributed under the terms of the         *)
+(*         *     GNU Lesser General Public License Version 2.1          *)
+(*         *     (see LICENSE file for the text of the license)         *)
+(************************************************************************)
+
+(** [load_init_vernaculars feed_ref opts ~state] Load vernaculars from
+   the init (rc) file *)
+val load_init_vernaculars : int ref -> Coqargs.coq_cmdopts -> state:Vernac.State.t-> Vernac.State.t
+
+(** [compile_files feed_ref opts] compile files specified in [opts] *)
+val compile_files : int ref -> Coqargs.coq_cmdopts -> unit
```

Comment:
The "feeeder" thing, that BTW I did not see becoming an integer, should not show up this way in the API.
A bare bone `int ref`?

Reply:
> The "feeeder" thing, that BTW I did not see becoming an integer, should not show up this way in the API.

How should it show then? [Hint you merged the PR :)] The integer is a `feeder_id` so you can detach it. There is no other clean way I'm afraid.

This is here due to improvements on init printing. I agree that it could use a cleanup, however I think such a cleanup goes beyond the scope of this PR so it will have to be done separately then.